bindings/go/src/fdb/errors.go

```diff
@@ -49,7 +49,9 @@ func (e Error) Error() string {
 // SOMEDAY: these (along with others) should be coming from fdb.options?
 
 var (
-	errNetworkNotSetup = Error{2008}
+	errNetworkNotSetup          = Error{2008}
+	errNetworkAlreadySetup      = Error{2009} // currently unused
```

Comment:
I guess the error code here should be consistent with https://github.com/jzhou77/foundationdb/blob/5d2deddb7dee53f10ece037be18a4f5d313cb049/flow/include/flow/error_definitions.h#L201, so probably we want to define the error there and use a different number here.

Reply:
My bad. Actually it's consistent.